src/mumble/MainWindow.cpp

```diff
@@ -225,6 +225,10 @@ void MainWindow::createActions() {
 	gsJoinChannel->setObjectName(QLatin1String("MetaChannel"));
 	gsJoinChannel->qsToolTip = tr("Use in conjunction with Whisper to.", "Global Shortcut");
 
+	gsListenChannel = new GlobalShortcut(this, idx++, tr("Listen Channel", "Global Shortcut"));
```

Comment:
```suggestion
	gsListenChannel = new GlobalShortcut(this, idx++, tr("Listen to Channel", "Global Shortcut"));
```
Or is that too long to fit in the UI? :thinking: